frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -417,10 +419,22 @@ const ManagePolicyPage = (managePoliciesPageProps: {
                      ${baseClass}__inherited-policies-button`}
                 onClick={toggleShowInheritedPolicies}
               >
-                {`${
-                  showInheritedPolicies ? "Hide" : "Show"
-                } inherited policies`}
+                {`${showInheritedPolicies ? "Hide" : "Show"} ${
+                  globalPolicies.length
+                } inherited ${
+                  globalPolicies.length > 1 ? "policies" : "policy"
+                }`}
```

Comment:
Would you mind creating a variable above the render for this? It's difficult to read.